basic-teal/app.R

```diff
@@ -1,10 +1,10 @@
 library(teal)
 
 app <- init(
+  data = list(IRIS = iris, MTCARS = mtcars),
   filter = teal_slices(
     teal_slice(dataname = "IRIS", varname = "Species", multiple = FALSE, locked = TRUE)
```

Comment:
`locked` is no longer a valid argument in `teal_slice()`, please remove this code.